Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/SqliteCentralRepo.java

```diff
@@ -832,48 +832,27 @@ public void processSelectClause(String selectClause, InstanceTableCallback insta
             releaseSharedLock();
         }        
     }      
-
-    @Override
-    public void executeInsertSQL(String insertSQL) throws CentralRepoException {
-        try {
-            acquireSharedLock();
-            super.executeInsertSQL(insertSQL);
-        } finally {
-            releaseSharedLock();
-        }
-    }
     
     @Override
-    public void executeSelectSQL(String selectSQL, CentralRepositoryDbQueryCallback queryCallback) throws CentralRepoException {
+    public void executeSqlAsPreparedStatement(String sql, List<Object> params) throws CentralRepoException {
         try {
             acquireSharedLock();
-            super.executeSelectSQL(selectSQL, queryCallback);
+            super.executeSqlAsPreparedStatement(sql, params);
```

Comment:
@raman-bt and @gdicristofaro, this method is documented as executing an UPDATE/INSERT/DELETE statement. We need to acquire the exclusive (write) lock, not the shared (read) lock.